capstone/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
           - 9200:9200
     worker:
         build: .
-        image: capstone:0.3.59-b5782fc50cfbbaa3c159988ca2e8abf1
+        image: capstone:0.3.60-1f51a4b03f405f7e3ef01a680aafaca1
```

Comment:
Oops, this has to be updated because of a parallel PR from Ben.

Reply:
Donesky